mautic/mautic
# Fixed popupBlocker JS message

Please answer the following questions. 

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | n
| Related user documentation PR URL | na
| Related developer documentation PR URL | na
| Issues addressed (#s or URLs) | #1666
| BC breaks? | n
| Deprecations? | n

**Note that all new features should have a related user and/or developer documentation PR in their respective repositories.** 

### Required
#### Description:

When creating a new email, sms, notification and others from campaigns, `response` wasn't defined leading to a JS error. This PR fixes this.

#### Steps to test this PR:
1. Ensure popups are blocked
2. Edit/create a campaign
3. Add a send email action
4. Click New Email button
5. Should get a JS alert stating that a popup blocker is in effect

### As applicable
#### Steps to reproduce the bug:
1. Same steps as above only a JS error will occur rather than getting the alert
